packages/datetime/src/_datepicker.scss

```diff
@@ -109,6 +109,15 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
     display: table-row;
   }
 
+  .DayPicker-WeekNumber {
+    display: table-cell;
+    vertical-align: middle;
+    text-align: center;
+    color: $pt-text-color-disabled;
+    font-size: $pt-font-size;
+    pointer-events: none;
```

Comment:
I don't think we want this property... it breaks any support for `onWeekClick`.
But in the interest of velocity I think we can merge this PR and I can investigate the `onWeekClick` issues separately.

Reply:
@giladgray can you file an issue to track?